scripts/local-cluster.sh

```diff
@@ -0,0 +1,439 @@
+#!/usr/bin/env bash
+
+# Contributor helpers to run local clusters
+
+set -euo pipefail
+
+HERE="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+ROOT="$(dirname "${HERE}")"
+
+log.info.no_newline() {
+  echo -en "[\e[34mck8s\e[0m] ${*}" 1>&2
+}
+log.info() {
+  log.info.no_newline "${*}\n"
+}
+log.warn.no_newline() {
+  echo -e -n "[\e[33mck8s\e[0m] ${*}" 1>&2
+}
+log.warn() {
+  log.warn.no_newline "${*}\n"
+}
+log.error.no_newline() {
+  echo -e -n "[\e[31mck8s\e[0m] ${*}" 1>&2
+}
+log.error() {
+  log.error.no_newline "${*}\n"
+}
+log.fatal() {
+  log.error "${*}"
+  exit 1
+}
+log.usage() {
+  log.fatal "$0 usage:
+  - config <name> <domain> <optional-ops-prefix> - configures a local cluster
+  - create <name> <config|flavor>                - creates a local cluster, optional args [ --skip-calico, --skip-minio ]
```

Comment:
I find it confusing when *flavor* is used twice (better in the doc where it distinguishes apps flavor from kind flavor) but I wonder if a different term could could be used?